static/js/redux/actions/timetable_actions.jsx

```diff
@@ -146,7 +147,7 @@ export const lockTimetable = (timetable) => (dispatch, getState) => {
   const state = getState();
 
   if (timetable.has_conflict) {
-    dispatch({ type: ActionTypes.TURN_CONFLICTS_ON });
+    dispatch(preferencesActions.turnConflictsOn());
```

Comment:
Thinking of getting rid of the preferences modal soon, what do you think?

Reply:
Yes, the preference modal is kinda buggy, I think we should remove

Reply:
Okay, let's do that after redux reducers are refactored